libraries/src/Component/ComponentHelper.php

```diff
@@ -252,37 +256,37 @@ public static function filterText($text)
 					$filter->blockedAttributes = $customListAttributes;
 				}
 			}
-			// Blacklists take second precedence.
-			elseif ($blackList)
+			// DisallowList take second precedence.
+			elseif ($disallowList)
 			{
-				// Remove the whitelisted tags and attributes from the black-list.
-				$blackListTags       = array_diff($blackListTags, $whiteListTags);
-				$blackListAttributes = array_diff($blackListAttributes, $whiteListAttributes);
+				// Remove the allowed tags and attributes from the disallowList.
+				$disallowedTags       = array_diff($disallowedTags, $allowedTags);
+				$disallowedAttributes = array_diff($disallowedAttributes, $allowedAttributes);
 
 				$filter = InputFilter::getInstance(
-					$blackListTags,
-					$blackListAttributes,
+					$disallowedTags,
+					$disallowedAttributes,
 					InputFilter::ONLY_BLOCK_DEFINED_TAGS,
 					InputFilter::ONLY_BLOCK_DEFINED_ATTRIBUTES
 				);
 
-				// Remove whitelisted tags from filter's default blacklist
-				if ($whiteListTags)
+				// Remove allowed tags from filter's default disallowList
```

Comment:
```suggestion
				// Remove allowed tags from filter's default Disallow List
```
Formalise the wording to show as separate words.